### PR TITLE
- Added image for tr preview ,- Fixed iframe video to be able to play in readme file

### PR DESCRIPTION
**Pull request by Moddinu**
- Added image for tr preview 
- Fixed iframe video to be able to play in readme file

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ ] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by SSwiniarski** [comment]
@Moddinu if we do not see movement on this PR in a couple of days we'll have to close this PR and unassign the issue.

**Comment by Moddinu** [comment]
I already pushed my code so you can close it once merged.

File: content/html/concepts/tables/tables.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ The table row element, denoted as `<tr>`, is used to display data within a row (
   </tr>
 </table>
 ```
+![source="https://www.w3schools.com/html/html_tables.asp"](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-tables-tr.png)
```
[the remark this answers is not in the document]
Comment (Moddinu) [reply content/html/concepts/tables/tables.md:45]: Done

File: content/html/concepts/tables/tables.md [hunk 2]
```diff
@@ -103,4 +105,4 @@ The table data element, denoted as `<td>`, is used to display individual data wi
 
 Watch this video for a step-by-step walkthrough on how to create an HTML table.
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/VjiD2EyJHIQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
[the remark this answers is not in the document]
Comment (Moddinu) [reply content/html/concepts/tables/tables.md]: Done reverted iframe back

File: content/html/concepts/tables/tables.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
----
+![image](https://github.com/Moddinu/docs/assets/5288834/b30bf57d-f69c-4fb3-8e3c-97943cf49ab5)---
```
Comment (SSwiniarski) [inline content/html/concepts/tables/tables.md:1]:
```suggestion
```
This is in the metadata, this is wrong.

File: content/html/concepts/tables/tables.md [hunk 4]
```diff
@@ -20,13 +20,8 @@ Generally, tables are used to display tabular data in particular. In the past, t
...
 ## Example
 
-|      Name      |    Location    |    Category     | Founded |        CEO        |               Info                |
-| :------------: | :------------: | :-------------: | :-----: | :---------------: | :-------------------------------: |
-|   Pied Piper   | Silicon Valley | Cloud Computing |  2014   | Richard Hendricks | A Middle-Out Compression Solution |
-|     Hooli      | Silicon Valley |   Enterprise    |  1997   |   Gavin Bensen    |       Hooli is About People       |
-| Raviga Capital | Silicon Valley | Venture Capital |  2012   |   Peter Gregory   |       Share Only in Success       |
-|     Aviato     | Silicon Valley |     Travel      |  2006   |  Erlich Bachman   |   Software Aggregation Program    |
-|    See Food    | Silicon Valley |     Mobile      |  2016   |     Jian-Yang     |        The Shazam of Food         |
+![image](https://github.com/Moddinu/docs/assets/5288834/1a865d50-a767-4736-8cc4-ff4d68e57615)
```
Comment (SSwiniarski) [inline content/html/concepts/tables/tables.md:23]: This image is in the wrong place. it needs to be uploaded to the `/media` folder and use the url:`https://raw.githubusercontent.com/Codecademy/docs/main/media/`

File: content/html/concepts/tables/tables.md [hunk 5]
```diff
@@ -93,11 +90,7 @@ The table data element, denoted as `<td>`, is used to display individual data wi
   </tr>
 </table>
 ```
-
-| Heading 1 | Heading 2 | Heading 3 |
-| :-------: | :-------: | :-------: |
-|  Cell 1   |  Cell 2   |  Cell 3   |
-|  Cell 4   |  Cell 5   |  Cell 6   |
+![image](https://github.com/Moddinu/docs/assets/5288834/2b15a2d2-d047-4426-a3bb-f04cfb137278)
```
Comment (SSwiniarski) [inline content/html/concepts/tables/tables.md:93]: This image is in the wrong place. it needs to be uploaded to the `/media` folder and use the url:`https://raw.githubusercontent.com/Codecademy/docs/main/media/`

File: content/html/concepts/tables/tables.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (SSwiniarski) [reply content/html/concepts/tables/tables.md:45]: You cannot use images from other websites, that's plagiarism an not allowed. You also put the url in the ALT text of the markdown and the actual image URL is pointing at the poor image you've uploaded to the media folder. Please remove that image.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Moddinu There are still several problems with this that need to be addressed.

**Comment by SSwiniarski** [comment]
@Moddinu are you still working on this? If we do not hear from you in another week we're going to close the PR and unassign the issue.

**Comment by Moddinu** [comment]
Please close the issue and unassign as did not find time to work on it.
Thank you learned alot

**Comment by SSwiniarski** [comment]
Closed at author request.

**Pull request closed without merging**